src/packages/excalidraw/CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ Please add the latest change on the top under the correct section.
 
 #### BREAKING CHANGE
 
+- Added support for disabling `image` tool (also disabling image insertion in general, though keeps support for importing from `.excalidraw` files) [#6320](https://github.com/excalidraw/excalidraw/pull/6320).
```

Comment:
Breaking change isn't a separate type but a subtype and this is interfering with continuity of the feature excalidraw API breaking change.
This should be added under feature and then breaking change as a subtype under it similar to `excalidrawAPI`

Reply:
moved to feature as filing under breaking change was a mistake. There's no breaking change with this API.

That said, we should be structuring our breaking changes differently, and if nothing else, list them first and that's the most important thing for host apps when reading a changelog.

Reply:
> That said, we should be structuring our breaking changes differently, and if nothing else, list them first and that's the most important thing for host apps when reading a changelog.

When breaking change is as result of some feature / fix then it should be a sub heading so its clear why it was introduced and how to deal with it instead of separating features/fixes and breaking change separately. Even when its a subheading its going to catch the attention as well and also explain the context better due to main heading